package.json

```
@@ -37,7 +37,7 @@
   "devDependencies": {
     "autoprefixer": "^10.4.21",
     "cross-env": "^7.0.3",
-    "hugo-extended": "0.147.5",
+    "hugo-extended": "0.147.6",
```

I'd suggest keeping it in sync with the Hugo version that 0.12.0 was published with:

```suggestion
    "hugo-extended": "0.147.5",
```